src/components/verification-grid/verification-grid.ts

```diff
@@ -933,6 +1141,110 @@ export class VerificationGridComponent extends WithShoelace(AbstractComponent(Li
     this.currentSubSelection = gridTiles.filter((tile) => tile.selected).map((tile) => tile.model);
   }
 
+  /**
+   * A common method that can be used to create consistent selection behavior
+   * across the different selection methods (click, alt, arrow, tab & highlight)
+   */
+  private processSelection(
+    tileIndices: number | number[],
+    { additive = false, toggle = false, range = false }: SelectionOptions = {},
+  ): void {
+    if (!this.canSubSelect()) {
+      return;
+    }
+
+    // If you want to set "singleDecisionMode", you will need to set it after
+    // selection has succeeded/failed.
+    this.singleDecisionMode = false;
+
+    // in desktop mode, unless the ctrl key is held down, clicking an element
+    // removes all other selected items
+    // while it is not possible to press the ctrl key on a tablet, the user can
+    // still overwrite the selection behavior using the selection-behavior
+    // attribute. Therefore, we have to check that we are not on explicitly
+    // using tablet selection mode.
+    if (this.userSelectionBehavior === "desktop" && !toggle && !additive) {
+      this.removeSubSelection();
+    }
+
+    const iterableIndices = Array.isArray(tileIndices) ? tileIndices : [tileIndices];
+    for (const tileIndex of iterableIndices) {
+      if (range) {
+        // if the user has never selected an item before, the multiSelectHead will be "null"
+        // in this case, we want to start selecting from the clicked tile
+        this.rangeSelectionHead ??= tileIndex;
```

Comment:
i think if in this case, if existing selection is none (head is null) it should just be treated as 0, and we should select from first tile to current tile index

Reply:
Fixed in

[`eafd840` (#430)](https://github.com/ecoacoustics/web-components/pull/430/commits/eafd8409c38c25e7e4522b356d317ebcb1e15357)